test/controllers/calls_controller_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'test_helper'
+
+class CallsControllerTest < ActionController::TestCase
+  include Devise::TestHelpers
+
+  def setup
+    @user = build :user
+    sign_in @user
+    @case = create :case
+    @call = build :call
+  end
+
+  describe 'create method' do
+    before do
+      @call = attributes_for :call
+    end
+
+    it 'should create and save a new call' do
+      assert_difference 'Case.find(@case).calls.count', 1 do
+        post :create, status: @call.status, id: @case
```

Comment:
You have the core ideas of this dead on. Way to go!

Before I merge I want to make sure that the post parameters are working the way I think they are -- rails used to be a real bear about testing parameters like you use `status` here, and I want to just double check, since we'll probably use this as a model for testing other controllers as well.